tests/functional/curate/cram/metadata-input.t

```diff
@@ -0,0 +1,43 @@
+Setup
+
+  $ pushd "$TESTDIR" > /dev/null
+  $ export AUGUR="${AUGUR:-../../../../bin/augur}"
+
+Testing metadata inputs for the curate command.
+Running the `normalize-strings` subcommand since it is the simplest subcommand,
+but this is not testing the functionality of the `normalize-strings` subcommand.
+See `normalize_strings.t` for the functionality tests of the subcommand.
```

Comment:
I guess I was treating the `normalize-strings` subcommand as this `passthru` subcommand. I can see how it would be helpful to have a subcommand that does not transform the data at all for troubleshooting.

Reply:
Added a `passthru` command in cdacc376efd3a6d5299d26eb7a911ebfe2819d9b and moved `normalize-strings` to a separate commit (ee7c155a5f1a570bf069e560fd1940c7671b6c58)